Updates/0237_rp_klaven_mortwake.sql

```diff
@@ -0,0 +1,201 @@
+DELETE FROM waypoint_path WHERE `PathId` = @PATH;
+INSERT INTO `waypoint_path` (`PathId`, `Point`, `PositionX`, `PositionY`, `PositionZ`, `WaitTime`, `ScriptId`) VALUES 
+(@PATH,1,-11137.872,556.66394,61.719536,0, 0),
+(@PATH,2,-11144.089,557.5722,61.71954,0, 0),
+(@PATH,3,-11150.329,555.53625,61.719536,0, 0),
+(@PATH,4,-11154.313,552.1036,61.765198,0, 0),
+(@PATH,5,-11156.641,544.77435,61.719536,0, 0),
+(@PATH,6,-11155.098,537.7074,61.719536,0, 0),
+(@PATH,7,-11149.677,531.89526,61.762802,0, 0),
+(@PATH,8,-11145.469,530.3515,61.762802,0, 0),
+(@PATH,9,-11142.669,529.8012,61.762802,0, 0),
+(@PATH,10,-11140.158,529.242,61.762802,0, 0),
+(@PATH,11,-11135.568,530.64386,61.762802,0, 0),
+(@PATH,12,-11131.786,533.37646,61.719536,0, 0),
+(@PATH,13,-11130.454,537.54376,61.719536,0, 0),
+(@PATH,14,-11129.113,544.04553,61.719536,0, 0),
+(@PATH,15,-11131.182,551.28595,61.71954,0, 0),
+(@PATH,16,-11133.829,554.5059,61.719536,0, 0);
+
+-- Defias Power Patrol
+-- Entry: 7052 Low: 7609870
+UPDATE `creature` SET `position_x`='-11131.519', `position_y`='540.82355', `position_z`='49.30846', `orientation`='2.80998' WHERE (`guid`='66991');
+
+SET @PATH := 7052; -- creatures
+DELETE FROM creature_movement_template WHERE `Entry` = @PATH;
+INSERT INTO `creature_movement_template` (`Entry`, `PathId`, `Point`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `WaitTime`, `ScriptId`) VALUES  
+(@PATH,0,1,-11131.658,549.87164,50.493183,100, 0, 0),
+(@PATH,0,2,-11136,554.8301,52.66208,100, 0, 0),
+(@PATH,0,3,-11140.383,556.89435,53.992172,100, 0, 0),
+(@PATH,0,4,-11144.682,556.6825,55.282417,100, 0, 0),
+(@PATH,0,5,-11149.117,554.4657,56.05738,100, 0, 0),
+(@PATH,0,6,-11152.269,552.6805,55.919537,100, 0, 0),
+(@PATH,0,7,-11144.682,556.6825,55.282417,100, 0, 0),
+(@PATH,0,8,-11140.383,556.89435,53.992172,100, 0, 0),
+(@PATH,0,9,-11136,554.8301,52.66208,100, 0, 0),
+(@PATH,0,10,-11131.658,549.87164,50.493183,100, 0, 0),
+(@PATH,0,11,-11130.728,546.13043,49.522034,100, 0, 0), 
+(@PATH,1,1,-11131.519, 540.82355, 49.395847,100, 1000, 0),
+(@PATH,1,2,-11131.519, 540.82355, 49.395847,2.80998, 10000, 705202);
+
+-- Entry: 7052 Low: 7609979
+UPDATE `creature` SET `position_x`='-11142.518', `position_y`='556.2102', `position_z`='36.04322', `orientation`='3.071779' WHERE (`guid`='66988');
+SET @PATH := 66988;
+DELETE FROM creature_movement WHERE `Id` = @PATH;
+INSERT INTO `creature_movement` (`Id`, `Point`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `WaitTime`, `ScriptId`) VALUES 
+(@PATH,1,-11142.518,556.2102,36.04322,3.228859,6000,0),
+(@PATH,2,-11149.39,554.4795,38.767456,100, 0, 0),
+(@PATH,3,-11152.168,551.7051,40.31828,100, 0, 0),
+(@PATH,4,-11154.249,547.9043,40.71954,100, 0, 0),
+(@PATH,5,-11152.688,546.27094,40.71954,100, 0, 0),
+(@PATH,6,-11150.508,546.6263,40.746307,100, 1000, 0),
+(@PATH,7,-11150.508,546.6263,40.746307,0.925024,5000,0),
+(@PATH,8,-11145.182,550.96234,44.038548,100, 0, 0),
+(@PATH,9,-11143.321,551.0007,44.9791,100, 0, 0), -- Custom d
```

Comment:
had to add 2 custom waypoints due to pathfinding issue

Reply:
whats with your comments at the end of the file?
what happens if this point is cut?

Reply:
That should have been removed before uploading, that was my reminder from yesterday about whats missing/wrong, cause i had to go sleep before finishing xd

https://youtu.be/C7-raZx2uZw 

here you can see it, he cant reach point 9 without a custom point before, same for point 34

Reply:
ok thanks for recording else i'd have to do it myself, legit point for me as otherwise it totally does not work, maybe add a comment for these points that are custom.

Reply:
Done.